src/services/vision/vision.ts

```diff
@@ -26,6 +33,22 @@ export interface Vision extends Resource {
   /**
    * Get a list of detections in the given image.
    *
+   * @example
+   *
+   * ```ts
+   * const camera = new VIAM.CameraClient(machine, 'my_camera');
+   * const vision = new VIAM.VisionClient(machine, 'my_vision');
+   *
+   * const mimeType = 'image/jpeg';
+   * const image = await camera.getImage(mimeType);
+   * const detections = await vision.getDetections(
+   *   image,
+   *   width,
+   *   height,
```

Comment:
Where are `width` and `height` coming from?

Reply:
good catch